README.md

```diff
@@ -1,8 +1,14 @@
-# depject
+# depject 
 
-minimal dependency injection
+> simplest dependency injection
```

Comment:
i often find simple is misunderstood for easy, so usually use minimal instead. but then again, simplest is more correct than minimal, since it fully injects all the dependencies. :wink: